x-pack/plugins/security_solution/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -402,9 +405,8 @@ export const signalRulesAlertType = ({
               wrapSequences,
             });
           }
-        } else {
-          throw new Error(`unknown rule type ${type}`);
```

Comment:
This is the old deprecated execution wrapper so doesn't really matter, but doesn't this change control flow by not `throw`'ing anymore? Or was this not needed in the first place as `params` validation would've caught unknown `types`'s?

Reply:
> Or was this not needed in the first place as params validation would've caught unknown types's?

Yup, that was just dead code.